app/services/google_policy_site.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class GooglePolicySite
+  BASE_URL = URI('https://policies.google.com').freeze
+
+  def self.locale_segment
+    active_locale = I18n.locale
+    active_locale == I18n.default_locale ? '' : "?hl=#{active_locale}"
+  end
+
+  def self.base_url
+    URI.join(BASE_URL, locale_segment).to_s
+  end
+
+  def self.privacy_url
+    URI.join(BASE_URL, "/privacy#{locale_segment}").to_s
+  end
+
+  def self.terms_url
+    URI.join(BASE_URL, "/terms#{locale_segment}").to_s
+  end
```

Comment:
This looks like its copied from our `MarketingSite` class, but I think that since here `local_segment` is query params, I think we could:
1. use `UriService.add_params`
2. redo a bnch
3. also add tests!

```suggestion
  def self.locale_params
    active_locale = I18n.locale
    active_locale == I18n.default_locale ? {} : { hl: active_locale }
  end

  def self.base_url
    UriService.add_params(BASE_URL, locale_params)
  end

  def self.privacy_url
    UriService.add_params(URI.join(BASE_URL, "/privacy"), locale_params)
  end

  def self.terms_url
    UriService.add_params(URI.join(BASE_URL, "/terms"), locale_params)
  end
```